spec/spec_helper.rb

```diff
@@ -29,7 +29,7 @@
   require 'simplecov'
 
   SimpleCov.root(File.expand_path('../..', __FILE__))
-  SimpleCov.formatter = Coveralls::SimpleCov::Formatter
+  SimpleCov.formatters = [Coveralls::SimpleCov::Formatter, SimpleCov::Formatter::HTMLFormatter]
```

Comment:
Sure, we can do the machine-readable output for Travis, and HTML output otherwise.